packages/ember-testing/lib/test/waiters.js

```diff
@@ -7,7 +7,8 @@ const callbacks = [];
 /**
    This allows ember-testing to play nicely with other asynchronous
    events, such as an application that is waiting for a CSS3
-   transition or an IndexDB transaction.
+   transition or an IndexDB transaction. The waiter runs periodically
```

Comment:
I would slightly clarify that it is ran periodically after async helpers (i.e. `click`, `andThen`, `visit`, etc) have executed prior to executing the next async step.